### PR TITLE
Disease categories for cmd codes in Puntland.

### PR DESCRIPTION
**Pull request by tomeksabala**

### CODE REVIEW

File: meerkat_frontend/src/files/pages/disease.html [hunk 1]
```diff
@@ -208,22 +208,29 @@
 
     function drawDiseaseTable(varID, locID) {
 
+        function draw_top_table(category_name, title) {
```
Comment (Gulfa) [inline meerkat_frontend/src/files/pages/disease.html]: Maybe we use a more descriptive name for this function. What is top table?

File: meerkat_frontend/src/files/pages/disease.html [hunk 2]
```diff
@@ -208,22 +208,29 @@
...
-                            dataObject = stripEmptyRecords(dataObject);
-
-                            drawTable('disease-table', dataObject, true, 'loadDisease');
-                        }
-                    });
-                });
+                category_name = data.name;
+                title = "ICD10 Sub-diseases";
+                draw_top_table(category_name, title);
+            } else if (data.id.indexOf('cmd_other') > -1) {
```
Comment (Gulfa) [inline meerkat_frontend/src/files/pages/disease.html]: I would suggest to just use else here. Then we can use this same approach if we want to group diseases in any other way later.

Comment (tomeksabala) [reply meerkat_frontend/src/files/pages/disease.html]: You're right. It's my bad. I should have written:
```
else if (data.id.indexOf('sub_category_") > -1)
```
This is open for any other sub categories extensions and not too generic to unnecessarily complicate current codeflow.

File: meerkat_frontend/src/files/pages/disease.html (hunk 1 above)
Comment (tomeksabala) [reply meerkat_frontend/src/files/pages/disease.html]: You got me here. Stale name, I've renamed it to reflect my best knowledge of what this reused piece of code does.

**Pull request merged**